CMake/stb/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.9)
+project(CheckSTB)
+include(GNUInstallDirs)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_library(alib SHARED alib.hpp alib.cpp)
+add_library(blib SHARED blib.hpp blib.cpp)
+
+#set(alib "alib.so")
+#set(blib "libblib.so")
+
+add_executable(CheckSTB
+  main.cpp
+  )
+
+target_link_libraries(CheckSTB
+  alib
+  blib
+  )
```

Comment:
I suppose that this is useful for local testing but we don't need to make it a part of the final commit---correct me if I'm wrong.  (I don't think this CMake configuration is used anywhere.)

Reply:
Not at all, I am sorry I did not even try it locally, I was working in an independent folder and then copied it to mlpack.
Just trying to verify with you if what I was doing make sense or not.

Reply:
No worries!  I think this is the right approach.  I'll play with it a bit more tomorrow. :+1: